app/services/integration_customers/hubspot_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module IntegrationCustomers
+  class HubspotService < ::BaseService
+    def initialize(integration:, customer:, subsidiary_id:, **params)
+      @customer = customer
+      @subsidiary_id = subsidiary_id
+      @integration = integration
+      @params = params&.with_indifferent_access
+
+      super(nil)
+    end
+
+    def create
+      create_result = create_service_class.call(
+        integration:,
+        customer:,
+        subsidiary_id: nil
+      )
+
+      return create_result if create_result.error
```

Comment:
I would rather use `unless create_result.success?` since we have this bool variable

Reply:
It is implemented exactly like this in all the other intergations. So we should change it there as well.

Reply:
Whay do you think? To leave is as is now, and refactor all the services in a separate PR or do it now? I'd choose another PR.